src/commands/service/list.ts

```diff
@@ -4,7 +4,7 @@ import {Instance} from 'mesg-js/lib/api'
 import Command from '../../root-command'
 
 export default class ServiceList extends Command {
-  static description = 'List all instances'
+  static description = 'List instances'
```

Comment:
```suggestion
  static description = 'List running services'
```
Let's no speak about instance